pkg/rbd/nodeserver.go

```diff
@@ -557,14 +608,12 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	if !notMnt {
 		return &csi.NodeExpandVolumeResponse{}, nil
 	}
-	imgInfo, devicePath, err := getDevicePathAndImageInfo(ctx, volumePath)
+
+	devicePath, err := getDevicePath(ctx, volumePath)
 	if err != nil {
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	if volName != imgInfo.ImageName {
```

Comment:
why this check is removed?

Reply:
As explained in https://github.com/ceph/ceph-csi/pull/758#issuecomment-573666885, the imageName is fetched from the omap now and in this context we don't have enough information (like pool and monitors) to fetch that information.